tests/runtime/builtin

```diff
@@ -108,6 +108,13 @@ EXPECT SUCCESS .*
 AFTER ./testprogs/uprobe_negative_retval
 TIMEOUT 5
 
+# BCC code it depends on is prone to race condition
+#NAME func_uprobe_exited_process
+#PROG uprobe:./testprogs/uprobe_symres_exited_process:test { print(func); exit(); }
+#EXPECT test
+#BEFORE ./testprogs/uprobe_symres_exited_process
+#TIMEOUT 5
```

Comment:
For other tests, we use
```
REQUIRES bash -c "exit 1"
```
to explicitly disable the test.

Reply:
Ah, nice, I was looking for a way to disable a test globally and I didn't find anything.